src/geo_templates/default/main_page/listing_auction.tpl

```diff
@@ -543,9 +543,11 @@
 	<!-- ADVERTISEMENT BEGIN -->
 	<h4 class="title">{$additional_text_3}</h4>
 	<div class="content_box_3 cntr banner">
-		<a href="http://www.kqzyfj.com/click-7066924-11191670" target="_blank"><img src="{external file='images/banners/1and1_300x100.jpg'}" alt="1and1 Hosting" title="1and1 Hosting" /></a>
-		<a href="http://www.hostmonster.com/track/geodesic/resources" target="_blank"><img src="{external file='images/banners/hostmonster_300x100.jpg'}" alt="HostMonster Hosting" title="HostMonster Hosting" /></a>
-		<a href="http://lunarpages.com/id/geodesic/goto/basic" target="_blank"><img src="{external file='images/banners/lunarpages_300x100.jpg'}" alt="Lunarpages.com Hosting" title="Lunarpages.com Hosting" /></a>
+        {* If you change the name of the sample banner below, be sure to update src\admin\getting_started_checks\design_removeDefaultBannersListingDetailsPage.php *}
+        {* NOTE: The above only pertains to Coders changing the software distro, not end users. *}

```

Comment:
Same here, we probably shouldn't put developer notes in the main part of templates that are meant to be customized for the site.